src/clientapi/index.js

```diff
@@ -118,18 +118,25 @@ exports.ClientAPI = Class(lib.PubSub, function () {
   import .UI;
   this.ui = new UI();
 
-
-  // this.track({
-  //  name: "campaignID",
-  //  category: "campaign",
-  //  subcategory: "id",
-  //  data: campaign
-  // });
-
   var map;
   try {
     if (GLOBAL.CACHE) {
       map = JSON.parse(GLOBAL.CACHE['spritesheets/map.json']);
+
+      // Add some defaults
+      for (var key in map) {
```

Comment:
if the spriter already outputs these keys, then why do we need to add them here? am i missing something - the spriter always creates map.json, right? or is there something else that sometimes creates it without these keys?